src/esp/bindings/SimBindings.cpp

```diff
@@ -45,6 +45,8 @@ void initSimBindings(py::module& m) {
                      &SimulatorConfiguration::sceneLightSetup)
       .def_readwrite("load_semantic_mesh",
                      &SimulatorConfiguration::loadSemanticMesh)
+      .def_readwrite("force_separate_semantic_scene_graph",
+                     &SimulatorConfiguration::forceSeparateSemanticSceneGraph)
```

Comment:
Maybe add a docstring explaining this?  Really need them for all of the config values, but perhaps this is a good place to start :)